lib/container.js

```diff
@@ -103,5 +103,9 @@ Container.prototype.ReceiverEvents = eventTypes.ReceiverEvents;
 Container.prototype.SenderEvents = eventTypes.SenderEvents;
 Container.prototype.SessionEvents = eventTypes.SessionEvents;
 Container.prototype.ConnectionEvents = eventTypes.ConnectionEvents;
+Container.prototype.DistributionMode = {
+    move: 'move',
+    copy: 'copy'
+};
```

Comment:
I would recommend to add DistributionMode in the following way, as that is how TS would transpile the enum object in TS to JS.
```js
var DistributionMode;
(function (DistributionMode) {
  /**
   * @property {string} move once successfully transferred over the link, the message will no longer be available
   * to other links from the same node.
   */
  DistributionMode['move'] = 'move';
  /**
   * @property {string} copy once successfully transferred over the link, the message is
   * still available for other links from the same node.
   */
  DistributionMode['copy'] = 'copy';
})(DistributionMode || (DistributionMode = {}));
Container.prototype.DistributionMode = DistributionMode;
```